packages/ui-extensions/src/surfaces/checkout/components/ScrollView/ScrollView.ts

```diff
@@ -24,7 +30,8 @@ export interface ScrollViewEvent {
 }
 
 export interface ScrollViewProps
-  extends BorderProps,
+  extends Pick<BackgroundProps, 'background'>,
```

Comment:
We should have the same thing for View as seen here. Its part of the same PR that 

https://github.com/Shopify/checkout-web/blob/main/packages/ui-extensions/src/surfaces/checkout/components/View/View.ts#L105

Reply:
Thanks @jffortier ! I'll add it back in 👍  I had removed that at @jamesvidler 's request, but I think he may have meant another file.

Reply:
Ya that was my bad, commented on the wrong file.

Reply:
`View.ts` is added back in now ✅ Thanks @jffortier and @jamesvidler !